.circleci/config.yml

```diff
@@ -30,3 +30,17 @@ workflows:
           filters:
             tags:
               only: /^v.*/
+
+      - architect/push-to-app-collection:
```

Comment:
Can you push this to all capi collections?

Reply:
I was not sure as the epic is about CAPA.
But yes I can :-)

Reply:
Sure but then we will create pr later on so might as well do it now